src/com/lftechnology/java/training/srijan/basics/genericclass/Srijan.java

```diff
@@ -0,0 +1,5 @@
+package com.lftechnology.java.training.srijan.basics.genericclass;
+
+public class Srijan {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty class, write its code or make it an "interface". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2094)